npc/re/merchants/new_insurance.txt

```diff
@@ -0,0 +1,104 @@
+//===== rAthena Script =======================================
+//= Insurance Exchanger
+//===== Description: =========================================
+//= Exchange items for New_Insurance item.
+//===== Changelogs: ==========================================
+//= 1.0 Initial release. [Balfear]
+//============================================================
+
+prontera,82,108,5	script	Heart Merchant#life01	4_M_NFDEADMAN2,5,5,{
```

Comment:
There is probably a `checkweight` since the new item will take an inventory slot, if someone can grab the text..!